.buildkite/pipelines/performance/daily.yml

```diff
@@ -17,6 +17,13 @@ steps:
     agents:
       queue: kb-static-ubuntu
     depends_on: build
+    key: tests
+
+  - label: ':shipit: Performance Tests dataset extraction for scalability benchmarking'
+    command: .buildkite/scripts/steps/functional/scalability_dataset_extraction.sh
+    agents:
+      queue: kibana-default
```

Comment:
We may want this on a dynamic agent to get a fresher cache..14 minutes for bootstrap.  @brianseeders do we have any best practices here?

Reply:
Probably try `queue: n2-2` here instead. `kibana-default` is really only meant for small, quick tasks (like under 60s).

Reply:
Bootstrap went down to [4.5 min](https://buildkite.com/elastic/kibana-single-user-performance/builds/1330#2d9281f5-9fe7-416f-b8df-1741798a7703) with `queue: n2-2`. Thank you for the tip!